imager_profile/models.py

```diff
@@ -24,76 +24,62 @@ class Meta:
         model = "PatronProfile"
 
     def get_queryset(self):
-        qs = super(PatronProfileManager, self).get_queryset()
-        return qs.filter(user__is_active=True)
+        '''retrn a queryset of active users'''
+        return User.objects.filter(is_active=True)
```

Comment:
This manager is on the `Photographer` model, so its `get_queryset` method should return a queryset of `Photographer` instances, not `User` instances.